src/patients/related-persons/AddRelatedPersonModal.tsx

```diff
@@ -45,15 +47,30 @@ const AddRelatedPersonModal = (props: Props) => {
 
   const onSearch = async (query: string) => {
     const patients: Patient[] = await PatientRepository.search(query)
-    return patients.filter((p: Patient) => p.id !== patient.id)
+    return patients.filter((p: Patient) => p.id !== patientId)
```

Comment:
we should pull patients from the `usePatients` hook

Reply:
After looking into this one, I'm not sure how to do it. Since hooks have to be called at the top level of the function, it can't be passed as part of a callback. Do you have any suggestions for how to incorporate the `usePatients` hook?

Reply:
if `onSearch` sets a state variable called `patientSearchQuery` (or something similar) and that state variable is used as part of `usePatients` search request, it should cause the patients to be refreshed when the search criteria changes.

Reply:
Thanks, I'll try that.